src/org/thoughtcrime/securesms/ConversationListActivity.java

```diff
@@ -101,12 +101,12 @@ protected void onCreate(Bundle icicle, boolean ready) {
     // it is not needed to keep all past update messages, however, when deleted, also the strings should be deleted.
     DcContext dcContext = DcHelper.getContext(this);
     DcMsg msg = new DcMsg(dcContext, DcMsg.DC_MSG_TEXT);
-    msg.setText("1.37 is out:\n\n"
+    msg.setText("1.38 is out:\n\n"
       + "📎 Removed upper size limit of attachments\n\n"
       + "🗜️ PNG images (screenshots!) are now compressed as well\n\n"
       + "⌨️ For Devs: Better apps for chats with sendToChat() and importFiles() (What is Webxdc? -> https://webxdc.org)\n\n"
       + "... and more at https://get.delta.chat/#changelogs");
```

Comment:
shouldn't this message go to the strings.xml file for translation? :thinking:

Reply:
maybe, however, we do not always localize these messages ... and that one seems quite technical anyways ...

Reply:
but yes, before weighten for and against, i just added a commit and make it translatable, still makes sense